### PR TITLE
Adjust code samples for the confirm widget and added information about modalClass option

### PR DESCRIPTION
**Pull request by serhiyzhovnir**
## Purpose of this pull request

This pull request (PR) adds code samples for the confirm widget and also it adds the information about modalClass option.
The PR is an adjusted copy of https://github.com/magento/devdocs/pull/4732 PR.
I've created it on purpose to not to increase repo size too much.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/widgets/widget_confirm.html
- https://devdocs.magento.com/guides/v2.2/javascript-dev-guide/widgets/widget_confirm.html
- https://devdocs.magento.com/guides/v2.1/javascript-dev-guide/widgets/widget_confirm.html

## Links to Magento source code

- https://github.com/magento/magento2/blob/2.1/app/code/Magento/Ui/view/base/web/js/modal/confirm.js#L17
- https://github.com/magento/magento2/blob/2.2/app/code/Magento/Ui/view/base/web/js/modal/confirm.js#L20
- https://github.com/magento/magento2/blob/2.3/app/code/Magento/Ui/view/base/web/js/modal/confirm.js#L20
- https://github.com/magento/magento2/blob/2.3/app/code/Magento/Checkout/view/frontend/web/js/sidebar.js#L90
- https://github.com/magento/magento2/blob/2.2/app/code/Magento/Checkout/view/frontend/web/js/sidebar.js#L90
- https://github.com/magento/magento2/blob/2.1/app/code/Magento/Checkout/view/frontend/web/js/sidebar.js#L74

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.1/javascript-dev-guide/widgets/widget_confirm.md [hunk 1]
```diff
@@ -109,18 +110,97 @@ The title of the confirmation window.
 
 **Default value**: `''`
 
+### `modalClass` {#confirm_modalClass}
+The CSS class of the confirm window.
```
Comment (jeff-matthews) [inline guides/v2.1/javascript-dev-guide/widgets/widget_confirm.md]: Add space between header and paragraph

File: guides/v2.1/javascript-dev-guide/widgets/widget_confirm.md [hunk 2]
```diff
@@ -109,18 +110,97 @@ The title of the confirmation window.
...
 - `confirm` callback: called when the confirmation button is clicked.
 - `cancel` callback: called when the cancel button is clicked.
-- `always` callback.
+- `always` callback: called when the popup is closed.
 
 ## Keyboard navigation {#confirm_key_navigation}
 
 The keyboard navigation for the alert windows is similar to the [navigation of the modal widget].
 
+## Code Sample
```
Comment (jeff-matthews) [inline guides/v2.1/javascript-dev-guide/widgets/widget_confirm.md]: Delete this line

File: guides/v2.1/javascript-dev-guide/widgets/widget_confirm.md [hunk 3]
```diff
@@ -109,18 +110,97 @@ The title of the confirmation window.
...
+
+### Code sample of initialization on an element
```
Comment (jeff-matthews) [inline guides/v2.1/javascript-dev-guide/widgets/widget_confirm.md]:
```suggestion
**Example 1**: initialization on an element
```

File: guides/v2.1/javascript-dev-guide/widgets/widget_confirm.md [hunk 4]
```diff
@@ -109,18 +110,97 @@ The title of the confirmation window.
...
+            cancel: function () {
+                // do something when the cancel button is clicked
+            }
+        }
+    });
+});
+</script>
+```
+
+### Code sample of standalone initialization
```
Comment (jeff-matthews) [inline guides/v2.1/javascript-dev-guide/widgets/widget_confirm.md]:
```suggestion
**Example 2**: standalone initialization
```

**Comment by rogyar** [comment]
Hi @jeff-matthews. Please, check the recent changes when you have time. Thank you!

**Comment by jeff-matthews** [comment]
@rogyar & @atwixfirster, the original changes modified the 2.1 file, which was symlinked across versions. Since we've moved 2.1 files to a new branch and stopped accepting 2.1 PRs, this PR needs to be updated. After merging with `master` recently, there are no changed files associated with this PR.

These changes need to be applied to the 2.2 file.

**Comment by rogyar** [comment]
Totally missed this point, thank you @jeff-matthews for the clarification. @serhiyzhovnir could you adjust your PR correspondingly, please?

**Comment by serhiyzhovnir** [comment]
Hi @jeff-matthews @rogyar
I have noticed that the code examples for confirm widget are already added.
It was done in the scope of https://github.com/magento/devdocs/pull/4802 PR.
The examples of https://github.com/magento/devdocs/pull/4802 PR is really similar to this one.
We have adjusted the existed code examples and added information about the modalClass option.
Thank you!

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**